src/lib/queryFromFilters.js

```diff
@@ -60,6 +60,8 @@ function addConstraint(query, filter) {
       query.greaterThan(filter.get('field'), filter.get('compareTo'));
       break;
     case 'containsString':
+      query.matches(filter.get('field'), filter.get('compareTo'), 'i');
```

Comment:
The documentation in [Parse.query](http://parseplatform.org/Parse-SDK-JS/api/classes/Parse.Query.html#methods_matches) states only two parameters: key and regex. What's the `'i'` string you pass here?